WordPress/Classes/ViewRelated/Blog/SiteSettingsViewController.m

```diff
@@ -97,6 +108,11 @@ - (void)viewDidLoad
     [super viewDidLoad];
 
     self.navigationItem.title = NSLocalizedString(@"Settings", @"");
+    if ([self.blog isHostedAtWPcom]) {
```

Comment:
Remove site should only be shown if `[self.blog supports:BlogFeatureRemovable]`
`SiteSettingsSectionAccount` should probably be shown if `self.blog.account == nil`